static/src/js/util/is-number.js

```diff
@@ -0,0 +1,9 @@
+const reg = new RegExp(/^\d+$/)
+
+/**
+ * Returns true or false if the string contains a number
```

Comment:
should this comment be `if the string **is** a number`? or should the method name change to `containsNumber`?